packages/rancher-istio/charts/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
I think `requirements.lock` and the binary in `charts/` should be omitted from this PR with a `.gitignore`

Reply:
fixed

Reply:
Removed the tgz actually failed the generate-charts CI step: 
> Error: found in Chart.yaml, but missing in charts/ directory: rancher-kiali-server

Reply:
Figured this out, the `helm dependency update` call doesn't get called for non package.yaml charts. Think we can just add that in after any sub-charts get prepared.

Reply:
Added a fix for this, although not entirely sure we want to go down that road. And another issue with crd generation getting double versions.

Reply:
I think this route makes sense; without the tgz files, developers won't be able to do `helm install` either on the chart. The solution when working with a chart that did have a `package.yaml` involved creating these tgz files and [ignoring them on a patch](https://github.com/rancher/charts/blob/f6f0c3ddbe3c56e62f3b9c32b4964a1c5fe230f5/scripts/generate-patch#L65), but this should have been done for all packages (see comments below).

For the `generate-charts` step, even if the tgz is removed when committing this change it should still be automatically generated during CI since we always run the `prepare` script before running `generate-charts`: https://github.com/rancher/charts/blob/f6f0c3ddbe3c56e62f3b9c32b4964a1c5fe230f5/Makefile#L10-L11

Reply:
Right, for these kind of charts the generate charts step won't work unless prepare is run first. It does make dev slightly harder but they can just run `helm dependency` manually to test un-committed changes.